### ISSUE TITLE
support linux？

### ISSUE THREAD

**Issue by sikenali** [johnmave126/wallpaper-switcher#2]

**Comment by johnmave126** [comment]
TLDR; No

The main reason to create this program is that all existing windows wallpaper changers use `IDesktopWallpaper` interface instead of `IActiveDesktop` interface. The former interface won't allow the animated transition one can get from Windows native slideshow. This is kinda the first one with both slideshow and transition (plus a mimic windows UI).
Adding support for Linux will be quite tricky. Different desktop managers may have different behaviors. Lack of animated transition may not even be a problem in the first place. And the windows UI doesn't really fit into Linux system.
I think there should be good enough Linux wallpaper manager out there so no support for Linux in foreseeable future.
